### ISSUE TITLE
Batch Normalizationについて理解したことをREADME.mdにまとめる

### ISSUE THREAD

**Issue by anuthys** [anuthys/batch-normalization#1]
[元論文](https://arxiv.org/abs/1502.03167)と[Qiitaの記事](https://qiita.com/search?q=batch+normalization)を参考にしながら、理解したことをまとめ、[`README.md`](https://github.com/anuthys/batch-normalization/blob/master/README.md)にまとめる。
